internal/output/output.go

```diff
@@ -0,0 +1,20 @@
+package output
+
+import (
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+)
+
+func CaptureAll() io.Writer {
+	tmpfile, err := ioutil.TempFile("", "tilt.output.*")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	os.Stdout = tmpfile
+	os.Stderr = tmpfile
+
+	return tmpfile
```

Comment:
oh! i remember my comment now. I meant that CaptureAll could return a different writer that didn't write to these temp files

Reply:
Aha I see!

Reply:
I think it's desirable to write to an actual file and keep it around. That way if there's an error that being swallowed we can still find it somewhere, even on a production build on a users machine.